packages/salesforcedx-utils-vscode/src/commands/commandletExecutors.ts

```diff
@@ -159,9 +167,16 @@ export abstract class LibraryCommandletExecutor<T>
       const success = await vscode.window.withProgress(
         {
           title: nls.localize('progress_notification_text', this.executionName),
-          location: vscode.ProgressLocation.Notification
+          location: vscode.ProgressLocation.Notification,
+          cancellable: this.cancellable
         },
-        () => this.run(response)
+        (progress, token) => {
+          token.onCancellationRequested(() => {
+            notificationService.showCanceledExecution(this.executionName);
+            return false;
```

Comment:
How does this `return false` work?

Reply:
I decided to remove this change for now, keeping this limited to just changing the method definition